docs/source/examples/subinplace.py

```diff
@@ -0,0 +1,27 @@
+#creating a linked model using sub in-place
+from gpkit import Variable, Model, LinkedConstraintSet, ConstraintSet
+from import_classes import M1, M2new
+
+#Create the two models
+m1 = M1()
+m2 = M2new()
+
+#create a list of submodes
+submodels = [m1, m2]
+
+#generate a constraint set for the full model
+constraints = ConstraintSet([submodels])
+
+#use sub in-place to change all y2 varkeys to y
+constraints.subinplace({"y2": "y"})
+
+#create a linked constraint set
+#now there is only a single y variable
+lc = LinkedConstraintSet(constraints)
```

Comment:
@bqpd, is this how we should be advocating to do this? I would have thought a `LinkedConstraintSet` was not necessary here - can we just subinplace and do no further linking?